pkg/smartcontract/param_context_test.go

```diff
@@ -31,8 +151,8 @@ var testCases = []struct {
 	},
 	{
 		input: `{"type":"Array","value":[
-{"type": "String", "value": "str 1"},
-{"type": "Integer", "value": 2}]}`,
+		{"type": "String", "value": "str 1"},
+		{"type": "Integer", "value": 2}]}`,
```

Comment:
Doesn't belong to 0e3e2179d7533f5b64b5934820657f2af7685717, it's a part of d8d99fe3d5b37918468eda87ee2aaf4b620bef3a.